mini-project/BillManagementSystem/accounting/schema/schema.py

```diff
@@ -0,0 +1,207 @@
+            success=True,
+            category=category
+        )
+    except models.Category.DoesNotExist as e:
+        return CategoryResponse(
+            success=False,
+            category= None,
+            error='Bro, this category Does not Exist!'
+        )    
+
+# User
+def get_all_users(root) -> List[User]:
+    users = models.User.objects.all()
+    users_list = list()
+    for user in users:
+        users_list.append(
+            User(
+            id=user.id,
+            username=user.user.username,
+            first_name=user.user.first_name,
+            last_name=user.user.last_name,
+            email=user.user.email,
+            birthday=user.birthday,
+            phone_number=user.phone_number,
+            bills = models.Bill.objects.filter(user=user.id),
+            individual_spendings = models.IndividualSpending.objects.filter(user=user.id)
+        )
+    )  
+    
+    return users_list
+
+# user
+def get_user_by_id(self, id: int) -> UserResponse:
+    try:
+        user : models.User = models.User.objects.get(id=id)
+        return User(
+            id=id,
+            username=user.user.username
+        )
+    except models.User.DoesNotExist as e:
+        return 
+    
+# Bill    
+def get_bills_by_filter(self, 
+                        id: Optional[int] = None, 
+                        user_id: Optional[int] = None, 
+                        title: Optional[str] = None, 
+                        category_id: Optional[int] = None) -> Bill:
+    
+    if not any((id, user_id, title, category_id)):
+        raise ValueError("Hey! Please query by at least one parameter!")
+    
+    if id is not None:
+        bill = models.Bill.objects.get(id=id)
+        return [bill]
+    else:
+        queryset = models.Bill.objects.all()
+        
+        if user_id:
+            user = models.User.objects.get(id=user_id)
+            bill = queryset.filter(user=user)
+            
+        if title:
+            bill = queryset.filter(title__icontains=title)
+            
+        if category_id:
+            category = models.Category.objects.get(id=category_id)
+            bill = queryset.filter(category=category)
+            
+        return bill
+
+@strawberry.type
+class Query:
+    category: CategoryResponse = strawberry.field(resolver=get_category_by_id)
+    categories: List[Category] = strawberry.field(resolver=get_all_categories)
+    user: List[User] = strawberry.field(resolver=get_all_users)
+    bill: List[Bill] = strawberry.field(resolver=get_bills_by_filter)
+
+@strawberry.type
+class Mutation:
+    @strawberry.mutation
+    def add_user(self, register_input: RegisterInput) -> AuthResponse:
+        django_user = AdminUser.objects.create_user(
+            username=register_input.username, 
+            email=register_input.email, 
+            password=register_input.password, 
+            first_name=register_input.first_name, 
+            last_name = register_input.last_name
+        )
```

Comment:
You can try doing the following, since you will be using all fields, and the names are the same as params. (You may need appropriate dict conversion and/or value changes):
```py
django_user = AdminUser.objects.create_user(
    **register_input
)

```

You can consider the following method for value extraction:
```py
interested_dict = {
    key: value
    for (key, value) in register_input.items()
    if key in ["username", "email", "password", "first_name", "last_name"]
}
```

Reply:
https://github.com/strawberry-graphql/strawberry-graphql-django/issues/235

Waiting for this bug to fix.